plugins/source/aws/resources/services/sns/subscriptions.go

```diff
@@ -33,6 +33,26 @@ func Subscriptions() *schema.Table {
 					PrimaryKey: true,
 				},
 			},
+			{
+				Name:     "delivery_policy",
```

Comment:
Instead of adding these, should unwrap fields in Transformer

Reply:
why unwrap? those were `json` before. I think this is a good way to override just by specifying the column directly.

Reply:
oh it's the type and not the missing fields? ok I misunderstood.